### DISCUSSION TITLE
Add the ability to create a subdocument that accesses a separate document

### DISCUSSION THREAD

**Discussion by AlexShku** [outline/outline#6757] [Ideas]
Hello! 
It is impossible to create a document if you have access to a separate document. It would be nice to add such a feature.

**Comment by AlexShku** [comment]
up

**Comment by tommoor** [comment]
To clarify, are you requesting that being given `write` access to a document also allows users to create child documents within?

**Comment by AlexShku** [comment]
Exactly! If a user has Editor rights, it would be nice to add the ability to create subdocuments.

**Reply by tommoor** [reply]
This is a tough one, document-level permissions should not extend outside of the document itself and I think there are many more usecases where this would be undesirable behavior.

Typically if you want folks to be able to contribute to the "structure" of the collection it would be better to give them collection access. There is also the workaround here of the document owner creating the sub-document

**Reply by AlexShku** [reply]
Understood! Thanks for the clarification, Tom!
